packages/cli/space.js

```diff
@@ -17,15 +20,36 @@ import * as Result from '@storacha/client/result'
  * @property {false} [caution]
  * @property {DIDMailto.EmailAddress|false} [customer]
  * @property {string|false} [account]
+ * @property {Array<{id: import('@ucanto/interface').DID, serviceEndpoint: string}>} [authorizeGatewayServices] - The DID Key or DID Web and URL of the Gateway to authorize to serve content from the created space.
+ * @property {boolean} [skipGatewayAuthorization] - Whether to skip the Gateway authorization. It means that the content of the space will not be served by any Gateway.
  *
  * @param {string|undefined} name
  * @param {CreateOptions} options
  */
 export const create = async (name, options) => {
   const client = await getClient()
   const spaces = client.spaces()
-
-  const space = await client.createSpace(await chooseName(name ?? '', spaces))
+  
+  let space
+  if (options.skipGatewayAuthorization === true) {
+    space = await client.createSpace(await chooseName(name ?? '', spaces), {
+      skipGatewayAuthorization: true,
+    })
+  } else {
+    const gateways = options.authorizeGatewayServices ?? []
```

Comment:
If this is an empty array does the default still get used?

Reply:
If empty it will throw an error from the `client.createSpace` function saying the user needs to provide the connections.